Translation/en/Text/Mods/Plugin - MoreStoreItems

```diff
@@ -0,0 +1,3 @@
+{0}は{1}でオナニーしている={0} is masturbating in {1}
+{0}は{1}でレズのセックスしている={0} is having lesbian sex in {1}
+エロ活動：{0}=Erotic activity in {0}
```

Comment:
Since this plugin pre-translates the strings before using they work like resource strings.  Add a comment mentioning this, so this isn't used as an example of how things work normally.

This does mean that if the items substituted in for `{0}` and `{1}` aren't already translated they won't be, where using an `sr:`  they would, but it shouldn't be an issue.

Since this plugin explicitly supports translation this should move to `Text/Plugins/KKS_MoreStoreItems/translation.txt` (and add `#enable fallback`).  This scopes the translations to the plugin.  See [Plugin-specific Manual Translations](https://github.com/bbepis/XUnity.AutoTranslator#plugin-specific-manual-translations) in the XUnity docs.

Reply:
Additional thought is that the plugin itself should be using `TranslationHelper.TryTranslate` rather than `TranslationHelper.TranslateAsync`.  `TryTranslate` will only apply local translations (like these) where `TranslateAsync` will try and auto-translate if it's not found in the manual translations, and that will almost certainly give back garbage due to the substitutions (which will now be in your translation cache).

Reply:
Scoping only works with IMGUI as far as I know.